clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -51,39 +51,34 @@ class AzureResourceManagerClient extends AzureBaseClient {
     super(subscriptionId)
   }
 
-  DeploymentExtended createLoadBalancerFromTemplate(AzureCredentials credentials,
-                                        String template,
-                                        String resourceGroupName,
-                                        String region,
-                                        String loadBalancerName) {
+  DeploymentExtended createResourceFromTemplate(AzureCredentials credentials,
+                                                String template,
+                                                String resourceGroupName,
+                                                String region,
+                                                String resourceName) {
     def parameters = [location : region]
-    createLoadBalancerFromTemplate(credentials,
-                                   template,
-                                   parameters,
-                                   resourceGroupName,
-                                   region,
-                                   loadBalancerName)
+    createResourceFromTemplate(credentials, template, parameters, resourceGroupName, region, resourceName)
   }
 
-  DeploymentExtended createLoadBalancerFromTemplate(AzureCredentials credentials,
```

Comment:
Does this overload for createResourceFromTemplate add value? Why not just create the parameters within this method?